src/comparers/compareNFTs.ts

```diff
@@ -178,29 +176,39 @@ export const compareNftFiles = (
   return diff;
 };
 
-export const compareNftGroupToDatabase = async (
-  modifiedNfts: NftData[],
+interface PostReturn {
+  warnings: string;
+  metadata: Map<Metadata>;
+}
+
+export const compareNftsToMetadataService = async (
+  modifiedNfts: Nft[],
   logger: Logger
 ): Promise<NftBatchDiff> => {
-  const nftDataForDatabase: { id: string }[] = [];
+  const nftDataForDatabase: { domainIds: string[] } = { domainIds: [] };
 
   for (const modified of modifiedNfts) {
-    nftDataForDatabase.push({
-      id: modified.id,
-    });
+    nftDataForDatabase.domainIds.push(modified.id);
   }
 
-  const databaseNfts = await getNftsFromDatabase(nftDataForDatabase, logger);
+  const metadataServiceUrl = env
+    .get("METADATA_SERVICE_URL")
+    .default("https://metadata-service-api.azurewebsites.net/api/metadata")
+    .asString();
 
-  const diff = compareNftGroups(databaseNfts, modifiedNfts, logger);
-  return diff;
-};
+  const serviceData: PostReturn = (
```

Comment:
Should there be some retry or error handling logic around this request?

Reply:
Added error handling for if response failed